docs/DeveloperGuide.md

```diff
@@ -495,6 +495,33 @@ testers are expected to do more *exploratory* testing.
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       - Expected: Similar to previous.
 
+### Command history
+1. Navigating through command history with the up/down arrow keys
+
+    1. Prerequisites: Enter several different commands (e.g., listemployees, addproject ProjectX, delete 1), so that there is a history of commands available.
+
+    1. Test case: Press the up arrow key (↑) in the command box.
+        - Expected: The previous command appears in the command box.
+       
+    1. Test case: Press the up arrow key (↑) repeatedly to cycle through older commands.
+        - Expected: Each press navigates one step back in the command history. The earliest command is reached when pressing the up arrow no longer changes the displayed command.
+   1. Test case: Press the down arrow key (↓) to navigate forward in the command history.
+        - Expected: Each press navigates one step forward in the command history. The most recent command is reached when pressing the down arrow no longer changes the displayed command.
```

Comment:
No, if you reach the most recent command and press down arrow again it will actually turn back to blank command box i think